x-pack/plugins/aiops/public/components/mini_histogram/mini_histogram.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+
+import { Chart, BarSeries, PartialTheme, ScaleType, Settings } from '@elastic/charts';
+
+import type { ChangePointHistogramItem } from '@kbn/ml-agg-utils';
+
+interface MiniHistogramProps {
+  chartData: ChangePointHistogramItem[];
+  label: string;
+}
+
+export const MiniHistogram: FC<MiniHistogramProps> = ({ chartData, label }) => {
+  const theme: PartialTheme = {
+    chartMargins: {
+      left: 0,
+      right: 0,
+      top: 0,
+      bottom: 0,
+    },
+    chartPaddings: {
+      left: 0,
+      right: 0,
+      top: 0,
+      bottom: 0,
+    },
+    scales: {
+      barsPadding: 0.1,
+    },
+  };
+
+  return (
+    <div
+      style={{
+        width: '80px',
+        height: '24px',
+        margin: '0px',
+      }}
+    >
+      <Chart>
+        <Settings theme={theme} showLegend={false} />
+        <BarSeries
+          id="Other"
```

Comment:
Super duper nit: this id should be lowercase to be consistent with others

Reply:
Good catch, the "Other" in this BarSeries was wrong anyway, changed it to `doc_count_overall`. Fixed in [959edc2](https://github.com/elastic/kibana/pull/136295/commits/959edc2984e33dcfc42538b347e2d21ed983b1fc).